.idea/php.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Was surprised to see we didn't .gitignore this. Understand the usefulness for project specific settings, but we've .dist files for the various tools in the repos, and the PHP version we can pick up more accurately from composer already. Do we .gitignore this file @fibble @sdh100shaun @gabrielg2020